components/post_view/post_list/post_list.jsx

```diff
@@ -159,19 +164,23 @@ export default class PostList extends React.PureComponent {
     }
 
     postsOnLoad = async (channelId) => {
-        if (this.props.focusedPostId) {
-            await this.props.actions.loadPostsAround(channelId, this.props.focusedPostId);
-        } else if (this.props.isFirstLoad) {
-            await this.props.actions.loadUnreads(channelId);
-        } else if (this.props.latestPostTimeStamp) {
-            await this.props.actions.syncPostsInChannel(channelId, this.props.latestPostTimeStamp, false);
+        const {focusedPostId, isFirstLoad, latestPostTimeStamp, isPrefetchingInProcess, actions} = this.props;
+        if (focusedPostId) {
+            await actions.loadPostsAround(channelId, this.props.focusedPostId);
+        } else if (isFirstLoad) {
+            if (!isPrefetchingInProcess) {
```

Comment:
Perhaps it would be better to still fire this request, regardless of prefetching, just in case the prefetching status gets stuck as in-progress somehow? It's a bit wasteful to make two requests for the same data if the user is switching channels while prefetching is happening, but I don't think it would cause us any real issues. 0/5 though.

Reply:
I don't see a possibility of `stuck as in-progress somehow`. It will either be morphed into success or failure. 

It is not likely that this happens but if a request fails it still will be the same way it used to be without the failure retires epic.